configure

```diff
@@ -657,6 +657,10 @@ def configure_node(o):
   elif target_arch in ('mips', 'mipsel'):
     configure_mips(o)
 
+  if flavor in ('aix'):
```

Comment:
I'd followed the style in the file just a few lines down, but given that we only expect one in the set the == 'aix' is probably better, which change